### ISSUE TITLE
Repo Link Question

### ISSUE THREAD

**Issue by ceetonice** [IBM/ibmi-oss-issues#26]
Hi,

Is the below still the best link for updating our repo's when I upgrade to V7R5. I'm seeing 7.2 references in my packages.

  https://public.dhe.ibm.com/software/ibmi/products/pase/rpms/ibm.repo

**Comment by markdirish** [comment]
Hi @ceetonice ,

Two things:

1. Since late 2021, we have shipped RPMs a little differently. You can read about it here: https://ibmi-oss-docs.readthedocs.io/en/latest/yum/IBM_REPOS.html.

To update, you should just have to install the new repo RPMs:

```sh
$ yum install ibmi-release ibmi-base
```

Then if you read the FAQ, it will tell you how to delete your existing `ibm.repo` file, since it is no longer needed.

2. You may see references to 7.2 in the RPM names. We use to build most RPMs on IBM i 7.2, so that is the earliest release that those RPMs work on. Don't worry, they still work on V7R5. There are some RPMs that may have specific 7.5 versions as well, but just know that a lower OS level in the rpm name doesn't preclude them from working on your system.

Hope that helps!

**Comment by ceetonice** [comment]
Thank you for the response. 

I should have gave more information. I created a custom repo that's hosted on one of our Linux boxes. Our IBMi repos are pointed to this Linux box. Previously the repo we added to our Linux box was https://public.dhe.ibm.com/software/ibmi/products/pase/rpms/ibm.repo. Is that still the correct one to use?

**Comment by kadler** [comment]
No. There is now a base repo and a version-specific repo. There is https://public.dhe.ibm.com/software/ibmi/products/pase/rpms/repo-base-7.3 and then eg https://public.dhe.ibm.com/software/ibmi/products/pase/rpms/repo-7.3 https://public.dhe.ibm.com/software/ibmi/products/pase/rpms/repo-7.4 etc

**Comment by ceetonice** [comment]
Thanks for all the help. I'm going to make that change.

**Comment by ceetonice** [comment]
Just for clarity. 

I should sync from both, correct? I see that repo-7.5 does not have all the packages

https://public.dhe.ibm.com/software/ibmi/products/pase/rpms/repo-base-7.3
https://public.dhe.ibm.com/software/ibmi/products/pase/rpms/repo-7.5

**Comment by markdirish** [comment]
Correct. 7.5 will only have the packages that we have built with specific 7.5 enhancements. All other packages are still built for the "base" release, which at this time is 7.3. So your system will have a mix of 7.5 and 7.3 packages on it (and some 7.2 that were built when _that_ was the base release)

**Comment by ceetonice** [comment]
Thanks!
